.github/workflows/flutter_ci.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
         run: |
           cargo install --force cargo-make
           cargo install --force duckscript_cli
+          cargo install --force cargo-ndk
```

Comment:
Is it necessary?

Reply:
When testing it would bypass cargo-ndk and not install it. It is a necessary tool to build Appflowy-core